libbeat/docs/gettingstarted.asciidoc

```diff
@@ -151,17 +151,17 @@ with your system:
 ["source","sh",subs="attributes,callouts"]
 ----------------------------------------------------------------------
 sudo apt-get install openjdk-7-jre
-curl -L -O https://download.elastic.co/logstash/logstash/packages/debian/logstash_{LS-version}-1_all.deb
-sudo dpkg -i logstash_{LS-version}-1_all.deb
+curl -L -O https://download.elastic.co/logstash/logstash/packages/debian/logstash_{LS-version}_all.deb
```

Comment:
Changed to match the final DEBs for this release. I'll just need to keep checking these versions for each release.